packages/common/src/PublicNetworks.js

```diff
@@ -43,9 +43,21 @@ const PublicNetworks = {
     daiAddress: "0xbF7A7169562078c96f0eC1A8aFD6aE50f12e5A99",
     wethAddress: "0xd0A1E359811322d97991E03f863a0C30C2cF029C"
   },
+  137: {
+    name: "polygon-matic",
+    etherscan: "https://explorer-mainnet.maticvigil.com/",
+    customTruffleConfig: {
+      confirmations: 2,
+      timeoutBlocks: 200
+    }
+  },
   80001: {
-    name: "mumbai",
-    etherscan: "https://explorer-mumbai.maticvigil.com/"
+    name: "polygon-mumbai",
+    etherscan: "https://explorer-mumbai.maticvigil.com/",
+    customTruffleConfig: {
+      confirmations: 2,
+      timeoutBlocks: 200
+    }
```

Comment:
just to be clear, i think this is ok, but this is more a meta comment about how to organize data for multi chain support. it seems like both chainids and networkids tend to [correlate](https://chainid.network/), and keying only by network id probably is fine even across chains.  I dont know of any instances where 2 chains would have the same network id, but i am paranoid of this.  Do you guys have any opinions on this?

Reply:
This is a really great point david, we might want to consder storing a chainID in this mapping of networkID's --> network configs, but I don't see anything to do rn